libs/libcommon/src/libcommon/config.py

```diff
@@ -40,72 +40,72 @@
 if TYPE_CHECKING:
     from libcommon.processing_graph import ProcessingGraphSpecification
 
-ASSETS_BASE_URL = "assets"
-ASSETS_STORAGE_DIRECTORY = None
-ASSETS_S3_FOLDER_NAME = "assets"
+ASSETS_BASE_URL = "http://localhost/assets"
```

Comment:
a lot of changes in the env vars. Could you update the README file?

In particular: I understand that the S3 bucket name must be passed as `ASSETS_STORAGE_ROOT`, right?

Reply:
> In particular: I understand that the S3 bucket name must be passed as ASSETS_STORAGE_ROOT, right?
Yes!
Sure! I totally forgot to add the new vars to the readme.